pkg/docker/docker.go

```diff
@@ -126,6 +126,62 @@ type RunContainerOptions struct {
 	CapDrop          []string
 }
 
+// asDockerConfig converts a RunContainerOptions into a Config understood by the
+// docker client library.
+func (rco RunContainerOptions) asDockerConfig() docker.Config {
+	return docker.Config{
+		Image: getImageName(rco.Image),
+		User:  rco.User,
+		// TODO why do we call runContainerTar with a "partial"
+		// docker.Config?
```

Comment:
runContainerTar doesn't do anything except flesh out the config and return it...